samples/app-anonymous-users/csharp/ClientApp/src/components/facebook-auth-end.jsx

```diff
@@ -0,0 +1,42 @@
+// <copyright file="facebook-auth-end.jsx" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// </copyright>
+
+import { useEffect } from "react";
+import * as microsoftTeams from "@microsoft/teams-js";
+
+const FacebookAuthEnd = props => {
+
+    useEffect(() => {
+        microsoftTeams.app.initialize();
+        getAuthToken();
+    });
+
+    // Get face book client side token.
+    const getAuthToken = () => {
+        microsoftTeams.app.initialize();
```

Comment:
You have two initializes

Reply:
actually there is 3 if you include the useEffect. Can you reduce this to one

Reply:
I have removed microsoftTeams.app.initialize() in the facebook-auth-end.jsx
Fixed.